boot/freeldr/freeldr/arch/uefi/uefimem.c

```diff
@@ -0,0 +1,295 @@
+/*
+ * PROJECT:     Freeldr UEFI Extension
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     UEFI Memory Managment Functions
+ * COPYRIGHT:   Copyright 2022 Justin Miller <justinmiller100@gmail.com>
+ */
```

Comment:
As in #5171, "FreeLoader UEFI Support" would look better for the PROJECT field, same for headers in other files ; Also, perhaps, you can omit "UEFI" in the PURPOSE fields, as it's obvious.